README.md

```diff
@@ -397,6 +397,41 @@ interface {
 }
 ```
 
+#### [Shutdown the Z-Wave API on the controller](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=shutdown)
+
+[compatible with schema version: 36+]
+
+```ts
+interface {
+  messageId: string;
+  command: "driver.shutdown";
+}
+```
+
+#### [Update driver options](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=updateoptions)
+
+[compatible with schema version: 36+]
+
+```ts
+interface {
+  messageId: string;
+  command: "driver.update_options";
+  options: EditableZWaveOptions;
+}
+```
+
+#### [Send test frame to node](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=sendtestframe)
+
+[compatible with schema version: 36+]
+
+```ts
+interface {
+  messageId: string;
+  command: "driver.send_test_frame";
+  options: EditableZWaveOptions;
```

Comment:
Are these the right options ?

Reply:
I see this in the code:
![CleanShot 2024-06-16 at 13 02 12@2x](https://github.com/zwave-js/zwave-js-server/assets/1444314/c6e306af-496b-41cb-b286-8717f6a4e389)